modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc

```diff
@@ -49,7 +49,7 @@ image::api-id.png[]
 . To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
 +
 `kubectl get svc -n <namespace>`
-. Verify that your microservice port is named following the requirements requested by Istio: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
+. Verify that your microservice port is named following the requirements requested by xref:http://istio.io/docs/setup/additional-setup/requirements/[Istio].
```

Comment:
Comment on service-mesh-cli-adapter-help.png: By requiring users to view a screenshot to read text, you're breaking accessibility standards, unless you also put that text in alt-text. In this case, that would be impossible. So, what is the purpose of this screenshot?

Reply:
Took the conversation offline.